BoxContentSDK/BoxContentSDK/Clients/BOXContentClient+Collection.h

```diff
@@ -39,6 +39,21 @@
  */
 - (BOXCollectionItemsRequest *)collectionItemsRequestWithID:(NSString *)collectionID
                                                     inRange:(NSRange)range;
+/**
+ *  Generate a request to retrieve the items in a collectio with classification information.
+ *
+ *  @param collectionID Collection ID.
+ *  @param metadataTemplateKey  The template key for specific metadata template
+ *  @param metadataScope    The scope of the metadata
+ *  @param range        An offset (NSRange location) and limit (NSRange limit). The maximum limit allowed is 1000.
```

Comment:
Nit: The added metadata parameters make this doc harder for me to interpret. Maybe we could mention above, in the general comment, that this retrieves a _page_ of items from a collection.

It might also be better to arrange the parameters with range second, since that is how they are in the request, but if we always put the range last, that also makes sense to me as a convention.